packages/rules/src/mustSatisfyPeerDependencies.ts

```diff
@@ -193,8 +191,12 @@ function checkSatisfyPeerDependencies(context: Context, opts: Options) {
   const allRequiredPeerDependencies: { [peerDependencyName: string]: IPeerDependencyRequirement[] } = {};
 
   // for each of this package's dependencies, add the dependency's peer requirements into `allRequiredPeerDependencies`
-  for (const [, dependencyNode] of packageNode.dependencies) {
-    const requiredPeerDependencies = dependencyNode.packageJson.peerDependencies;
+  for (const dependency of Object.keys(packageDependencies)) {
+    const dependencyPackageJsonPath = require.resolve(`${dependency}/package.json`, {
```

Comment:
i think it's possible for this to throw, if your node_modules get into a weird state. should we catch the error and move on?

Reply:
we weren't doing this previously, and I haven't heard of any issues yet
https://github.com/monorepolint/monorepolint/blob/78e8606caa32c9882828bddec760f715519b2628/packages/rules/src/util/packageDependencyGraphService.ts#L44

I'm happy to make the change if you'd like, but also happy to leave as is :D

Reply:
cool, if it's existing behavior then i think it's fine